config/WarpDrive/loot-default.xml

```diff
@@ -65,6 +65,18 @@
 		<loot item="ThermalFoundation:material"	damage="66"		   minQuantity="1" maxQuantity="4" weight="25" />
 		<loot item="ThermalFoundation:material"	damage="73"		   minQuantity="1" maxQuantity="4" weight="45" />
 
+		<!-- Adding Thermal equal loot w/ same probability -->
+
+		<loot item="ThermalDynamics:ThermalDynamics_0"       minQuantity="1" maxQuantity="2"  weight="18" /><!-- Leadstone Fluxduct -->
+		<loot item="ThermalDynamics:ThermalDynamics_0"     damage="1"  minQuantity="1" maxQuantity="2"  weight="19" /><!-- Hardened Fluxduct -->
```

Comment:
It would be better to give them the material to make the pipes like you did with the Electrum Ingot.
FluxDucts are cheap, especially if you are in space already. 

No need to be stingy with loot here since you access the dimension with lategame tech.

Reply:
I would counter-argue, IC2 wires are just as cheap and are given as their "full" state. So why not TE?
Especially if you are this deep in IC2 to go to space.